src/views/shared/ScssContainer.js

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+    'okta'
+  ],
+  function (Okta) {
+    return Okta.View.extend({
+      className: 'auth-container-container',
```

Comment:
Do we need to create a new top level div? Ideally we would use our current #okta-sign-in one (which is already supposed to be the root div that we apply all styles to).